crates/supervisor/core/src/syncnode/node.rs

```diff
@@ -72,45 +77,75 @@ impl ManagedNode {
     ///
     /// Analyzes the event content and takes appropriate actions based on the
     /// event fields.
-    /// TODO: Call relevant DB functions to update the state
-    pub fn handle_managed_event(event_result: Option<ManagedEvent>) {
+    async fn handle_managed_event(
+        event_tx: &mpsc::Sender<NodeEvent>,
+        event_result: Option<ManagedEvent>,
+    ) {
         match event_result {
             Some(event) => {
-                debug!(target = "managed_node", event = ?event, "Handling ManagedEvent");
+                debug!(target: "managed_node", ?event, "Handling ManagedEvent");
 
                 // Process each field of the event if it's present
                 if let Some(reset_id) = &event.reset {
-                    info!(target = "managed_node", reset_id = %reset_id, "Reset event received");
+                    info!(target: "managed_node", ?reset_id, "Reset event received");
                     // Handle reset action
                 }
 
                 if let Some(unsafe_block) = &event.unsafe_block {
-                    info!(target = "managed_node", ?unsafe_block, "Unsafe block event received");
-                    // Handle unsafe block
+                    info!(target: "managed_node", ?unsafe_block, "Unsafe block event received");
```

Comment:
same here, any reason for using debug formatting? otherwise use display formatting as in suggestion. see docs for `tracing` library 
```suggestion
                    info!(target: "managed_node", %unsafe_block, "Unsafe block event received");
```